gl-client: wildcard import of `ring` crate leads to compilation errors

The `gl_client` crate depends on several crates using a wildcard for the version. Is there a reason to do this? The latest version of `ring` introduced some API changes, leading to build errors.

Example:
```
error[E0061]: this function takes 3 arguments but 2 arguments were supplied
   --> .../gl-client/src/node/service.rs:41:15
    |
41  |         match EcdsaKeyPair::from_pkcs8(&signature::ECDSA_P256_SHA256_FIXED_SIGNING, key.as_ref()) {
    |               ^^^^^^^^^^^^^^^^^^^^^^^^----------------------------------------------------------- an argument of type `&dyn SecureRandom` is missing
    |
note: associated function defined here
   --> .../ring-0.17.0/src/ec/suite_b/ecdsa/signing.rs:105:12
    |
105 |     pub fn from_pkcs8(
    |            ^^^^^^^^^^
help: provide the argument
    |
41  |         match EcdsaKeyPair::from_pkcs8(&signature::ECDSA_P256_SHA256_FIXED_SIGNING, key.as_ref(), /* &dyn SecureRandom */) {
    |                
```

--- a/libs/gl-client/Cargo.toml
+++ b/libs/gl-client/Cargo.toml
@@ -9,20 +9,20 @@ permissive = []
 export = ["chacha20poly1305", "secp256k1"]
 
 [dependencies]
-anyhow = "*"
+anyhow = "1.0.75"
 base64 = "^0.21"
 bech32 = "0.9.1"
 bytes = "1.2.1"
-hex = "*"
+hex = "0.4.3"
 http = "0.2"
 http-body = "^0.4"
 lightning-invoice = "0.24.0"
 log = "^0.4"
-pin-project = "*"
+pin-project = "1.1.3"
 prost = "0.11"
 reqwest = {version="^0.11", features=["json", "rustls-tls-native-roots"], default-features = false}
-ring = "*"
-rustls-pemfile = "*"
+ring = "~0.16.20"
+rustls-pemfile = "1.0.3"
 sha256 = "1.1.4"
 tokio = { version = "1", features = ["full"] }
 tonic = { version = "^0.8", features = ["tls", "transport"] }

